.github/workflows/osv-scanner-unified-action.yml

```diff
@@ -49,3 +49,4 @@ jobs:
         --skip-git
         ./
         ./docs/
+      fail-on-vuln: false
```

Comment:
actually one question: what's the motivation for false in this case?

Reply:
Ah, good catch!, it shouldn't be false, I accidentally left that in from me testing this workflow. Removed now.